ChangeLog

```diff
@@ -1,3 +1,798 @@
+    - Update Long Long Way volcanoes minable areas
+- Rephrase some map hints (#4211)
+- Update Europa Map to v1.2 (#4506)
+- Remove version numbers from all maps' file names
+- Adjust field size for mountains (#4562)
+- v1.0 Maps Review (#4634)
+    - New map Lesser Ring
+    - Updated map Wide World
+    - Deleted map Sun of Fire
+    - Marked Desert Tournament as unbalanced and renamed it
+- Adapt maps' minimum version number (#4646)
+
+
+### Saveloading
+
+- Replace unknown programs with "work" while saveloading (#3961)
+- Add some scenario savegame compabitility (#3963)
+    - Scenario tribes
+    - reveal_scenario
+- Rip out savegame compatibility code (#3980)
+- Fix map compatibility for MapObject Loaders (#4028)
+- Fix game saving crash in spectator mode (#4095)
+- Speedup player vision packet (#4113): Combine fields data into long strings
+- Skip redundant information in view packet (#4144)
+- Fix loading critters + resources
+    - Fix loading critters from map + fix an example
+    - Fis resources in lua_bases
+- Write timestamps in zip filesystem (#4193)
+- Fail gracefully on Lua PANIC (#4147)
+- Fix description loading in LuaBases (#4182)
+- Fix the `stol` saveloading issue (#4387)
+- Rediscover kVisible fields when loading the game (#4395)
+- Fix Lua savegame compatibility (#4437)
+- Unify legacy lookup tables (#4415)
+    - Merge `WorldLegacyLookupTable` and `TribesLegacyLookupTable` into `DescriptionsCompatibilityTable`.
+    - Let `Descriptions` handle the lookup to make the saveloading code easier to read
+- Fix savegame compatibility #4482 (#4483)
+- Saveload lua textdomain stack (#4461): Persist textdomain stack in scripting packet
+- Fix saveloading in economy options window (#4498)
+- Fix loading saves with see_all players (#4525)
+- Saveload `TrainingSite::checked_soldier_training_` to fix a crash (#4524)
+- Delete a dead function that causes savegame loading errors
+- Allow savegames anywhere (#4515)
+- Increase player info packet version (#4581)
+- Warn about replay version mismatch (#4605)
+- Fix S2 map loader to load stones correctly (#4619)
+- Add missing postload call for replays (#4616)
+- Hotfix for savegame compatibility #4273 (#4706)
+- Immovable savegame compatibility (#4690)
+- Reduce crashes from missing add-on dependencies (#4710)
+- Don't store Script add-ons in savegames (#4811)
+
+
+### AI
+
+- Fix a crash due to wrong AI Hint (#4191)
+- Give the AI its own namespace (#4225)
+- Quick AI fix to avoid crashes (#4388)
+- AI quick fix (#4401)
+- Read mining information for help & AI (#4489)
+    - Read AI mining hints from production program
+    - Add building name to deprecation messages
+    - Automate mining percentages in building help
+    - mines_percent is unused, so removing it
+- Trainingsites only evaluate fish/meat substitutes if present in inputs (#4641)
+- Delay AI initalization after scenario init (#4656)
+- Fixed an unsafe AI time operation (#4749)
+- AI production support (#4618)
```

Comment:
sorry for being late but this was such a big change that it would fit better with the bullet points from #4618 and #4465 as sub points,